src/main/java/organice/model/Model.java

```diff
@@ -57,6 +58,11 @@
      */
     boolean hasPerson(Person person);
 
+    /**
+     * Returns true if a doctor in charge with the same nric as {@code doctorInCharge} exists in the address book.
```

Comment:
Change address book to ORGANice.

Reply:
not necessary, we have a different PR for that alr

Reply:
Noted